### PR TITLE
docs: Sapphire localnet references consolidated page under tools

### PR DESCRIPTION
**Pull request by aefhm**
## Description

Necessary for https://github.com/oasisprotocol/docs/issues/959. Blocked by https://github.com/oasisprotocol/docs/pull/977, and https://github.com/oasisprotocol/cli/pull/310.

### CODE REVIEW

File: docs/guide.mdx [hunk 1]
```diff
@@ -72,6 +72,13 @@ ParaTime.
 [network-parameters]: https://github.com/oasisprotocol/docs/blob/main/docs/node/mainnet/README.md
 [Testnet]: https://github.com/oasisprotocol/docs/blob/main/docs/node/testnet/README.md
 
+## Localnet
+
+For development and testing, you can run a local [instance][localnet] of the
+entire Sapphire stack.
+
+[localnet]: /dapp/tools/localnet
```
Comment (aefhm) [inline docs/guide.mdx:80]:
```suggestion
[localnet]: ../tools/localnet.mdx
```

File: docs/network.mdx [hunk 2]
```diff
@@ -17,7 +17,7 @@ import {AddSapphireToMetaMask as S, AddSapphireTestnetToMetaMask as ST} from '@s
 
 
 [faucet]: https://faucet.testnet.oasis.io/
-[localnet]: https://github.com/oasisprotocol/oasis-web3-gateway/tree/main/docker
+[localnet]: /dapp/tools/localnet
```
Comment (aefhm) [inline docs/network.mdx:20]:
```suggestion
[localnet]: ../tools/localnet.mdx
```

**Comment by aefhm** [comment]
Build requires https://github.com/oasisprotocol/oasis-sdk/pull/2023

**Review by rube-de — APPROVED** [review]
looks good for me

File: docs/guide.mdx [hunk 3]
```diff
@@ -72,6 +72,13 @@ ParaTime.
 [network-parameters]: https://github.com/oasisprotocol/docs/blob/main/docs/node/mainnet/README.md
 [Testnet]: https://github.com/oasisprotocol/docs/blob/main/docs/node/testnet/README.md
 
+## Localnet
+
+For development and testing, you can run a local [instance][localnet] of the
+entire Sapphire stack.
+
+[localnet]: ../tools/localnet.mdx
```
Comment (rube-de) [inline docs/guide.mdx:80]: needs  full path, as it's outside of sapphire

**Review by rube-de — CHANGES_REQUESTED** [review]

File: docs/guide.mdx (hunk 3 above)
Comment (aefhm) [reply docs/guide.mdx:80]:
```suggestion
[localnet]: https://github.com/oasisprotocol/docs/blob/main/docs/dapp/tools/localnet.mdx
```

File: docs/network.mdx [hunk 4]
```diff
@@ -17,7 +17,7 @@ import {AddSapphireToMetaMask as S, AddSapphireTestnetToMetaMask as ST} from '@s
 
 
 [faucet]: https://faucet.testnet.oasis.io/
-[localnet]: https://github.com/oasisprotocol/oasis-web3-gateway/tree/main/docker
+[localnet]: ../tools/localnet.mdx
```
Comment (aefhm) [inline docs/network.mdx:20]:
```suggestion
[localnet]: https://github.com/oasisprotocol/docs/blob/main/docs/dapp/tools/localnet.mdx
```

**Review by rube-de — APPROVED** [review]

**Pull request merged**